### PR TITLE
Add additional resources from Sandy - resolves #49

### LINKED ISSUE #49: do we want to add Sandy's list of resources to the appendix?

### PR DESCRIPTION
**Pull request by ehumph**
This PR adds additional resources to the Appendix and resolves issue #49. These resources are from Beth Levine at the Arnold Library.

### CODE REVIEW

**Comment by ehumph** [comment]
Resolves #49

File: Appendix.Rmd [hunk 1]
```diff
@@ -5,6 +5,22 @@
...
+
+Checklist for Researchers 
+
+Policy Readiness Checklist for Librarians 
+
+Example DMSP & Example DMSP catalog 
+
+You can download the materials and see any updated releases at our [OSF project page](https://osf.io/uadxr/) . They have also utilized the OSF wiki to provide descriptions of our outputs, as well as citation information. 
+
+We hope that you will spread the word and use these resources at your institution and in your libguides. Please feel free to remix these documents to suit your needs with credit to the library working group. Please contact Beth Levine with any questions.
```
Comment (avahoffman) [inline Appendix.Rmd:22]: how should one contact Beth?

File: Appendix.Rmd [hunk 2]
```diff
@@ -5,6 +5,22 @@
 
 The official NIH DMS Policy FAQ page can be found here: [`https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/`](https://osp.od.nih.gov/scientific-sharing/nih-policy-for-data-management-and-sharing-faq/){target="_blank"}.
 
+# Additional resources {-}
+
+Librarians and other stakeholders have joined minds to produce guidance materials for researchers, libraries, and librarians to prepare for the 2023 NIH Data Management and Sharing Policy. The working group at the Arnold Library has created documents for librarians and researchers to use in their institutions as they work to comply with the new policy, including: 
+
+Glossaries of DMSP & NIH Grant Terminology
```
Comment (avahoffman) [inline Appendix.Rmd:12]: maybe make this a bulleted list?

File: Appendix.Rmd [hunk 3]
```diff
@@ -5,6 +5,22 @@
...
+
+Glossaries of DMSP & NIH Grant Terminology 
+
+Checklist for Researchers 
+
+Policy Readiness Checklist for Librarians 
+
+Example DMSP & Example DMSP catalog 
+
+You can download the materials and see any updated releases at our [OSF project page](https://osf.io/uadxr/) . They have also utilized the OSF wiki to provide descriptions of our outputs, as well as citation information.
```
Comment (avahoffman) [inline Appendix.Rmd:20]:
```suggestion
You can download the materials and see any updated releases at our [OSF project page](https://osf.io/uadxr/). They have also utilized the OSF wiki to provide descriptions of our outputs, as well as citation information. 
```

Comment (avahoffman) [reply Appendix.Rmd:20]: Can we link to the OSF wiki?

**Comment by ehumph** [comment]
@avahoffman I added the Wiki link as well as some links to webpages and guides on data management other academic institutions have put together

**Review by avahoffman — APPROVED** [review]
LGTM!

**Pull request merged**